end-to-end-tests/specs/genome-nexus/mutationTable.spec.js

```diff
@@ -81,4 +81,46 @@ describe('Mutation Table', function() {
         });
     });
 
+    describe('try getting GNOMAD from genome nexus', ()=>{
+        before(()=>{
+            var url = `${CBIOPORTAL_URL}/results/mutations?Action=Submit&RPPA_SCORE_THRESHOLD=2.0&Z_SCORE_THRESHOLD=2.0&cancer_study_list=nsclc_tcga_broad_2016&case_set_id=nsclc_tcga_broad_2016_cnaseq&data_priority=0&gene_list=TP53&geneset_list=%20&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=nsclc_tcga_broad_2016_cna&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=nsclc_tcga_broad_2016_mutations&tab_index=tab_visualize`;
+
+            goToUrlAndSetLocalStorage(url);
+            // mutations table should be visiable after oncokb icon shows up,
+            // also need to wait for mutations to be sorted properly
+            browser.waitForVisible("tr:nth-child(1) [data-test=oncogenic-icon-image]",300000);
+        });
+
+        it('should show the gnomad table after mouse over the frequency in gnomad column', ()=>{
+    
+            browser.waitForText('//*[text()="TCGA-78-7540-01"]',30000);
+            // show the gnomad column
+            browser.scroll(1000, 0);
+            // click on column button
+            browser.click("button*=Columns");
+            // scroll down to activated "GNOMAD" selection
+            browser.scroll(1000, 1000);
+            // click "GNOMAD"
+            browser.click('//*[text()="GNOMAD"]');
+            // find frequency
+            const frequency = '[data-test2="TCGA-78-7540-01"][data-test="gnomad-column"]';
+            browser.waitForExist(frequency, 30000);
+            // wait for gnomad frequency show in the column
+            browser.waitUntil(() => {
+                var textFrequency = browser.getText(frequency);
+                return textFrequency.length >= 1;
+            }, 600000, "Frequency data not in Gnoamd column");
+            // check if the column has <0.0001 
+            assert.equal(browser.getText(frequency), '<0.0001');
```

Comment:
@leexgh this prolly needs to be updated (no longer showing that)

Reply:
@inodb sure will do